edward/stats/distributions.py

```diff
@@ -9,114 +9,83 @@ class Distribution:
     """Template for all distributions."""
     def rvs(self, size=1):
         """
-        Parameters
-        ----------
-        size : int, list, or tuple, optional
-            Number of samples, in a particular shape if specified in a
-            list or tuple with more than one element.
-
-        params : float or np.array
-
         Returns
         -------
         np.ndarray
-            np.array of dimension (size x shape), where shape is the
-            shape of its parameter argument. For multivariate
-            distributions, shape may correspond to only one of the
-            parameter arguments, e.g., alpha in Dirichlet, p in
-            Multinomial, mean in Multivariate_Normal.
+            size-dimensional vector; scalar if size=1
```

Comment:
i moved the return type description into each class that derives from `Distribution`. i give the univariate description for univariate distributions, and the multivariate description for multivariate distributions.

i'll reintroduce the general one though, i don't mind.